OpenPNM/Network/__GenericNetwork__.py

```diff
@@ -415,6 +418,9 @@ def find_neighbor_throats(self, pores, mode='union', flatten=True):
         >>> pn.find_neighbor_throats(pores=[0, 1],flatten=False)
         array([array([0, 1, 2]), array([0, 3, 4, 5])], dtype=object)
         """
+        pores = sp.array(pores,ndmin=1)
+        if sp.size(pores) == 0:
+            return sp.array([],ndmin=1)
```

Comment:
This line and 421 don't follow pep8 either. To test locally, run `py.test --pep8`